apps/app/components/AppMenu/index.tsx

```diff
@@ -137,6 +138,12 @@ function AppMenu({ hidePrimaryMenu, collapsed }: any) {
                 <SubMenu icon={<CalendarOutlined />} title="Eventos">
                   <Item key="/events">Listar eventos</Item>
                   <Item key="/admin/event">Criar evento</Item>
+                  <Item key="/admin/lectures">Listar sessões</Item>
+                  <Item key="/admin/lectures/new">Criar sessão</Item>
+                </SubMenu>
+                <SubMenu icon={<UsergroupAddOutlined />} title="Sessões">
+                  <Item key="/admin/lectures">Listar sessões</Item>
+                  <Item key="/admin/lectures/new">Criar sessão</Item>
```

Comment:
I don't think it makes sense to have this buttons (list and create lecture) on both menus